README.md

```diff
@@ -116,7 +116,7 @@ to convert invalid characters to valid ones.
 
 ```php
 $generator->generate('Damn 💩!!');                                           // Result: damn
-$generator->generate('Damn 💩!!', ['transforms' => ['💩 > Ice-Cream']]);     // Result: amn-ce-ream
+$generator->generate('Damn 💩!!', ['transforms' => ['💩 > Ice-Cream']]);     // Result: damn-ce-ream
```

Comment:
It’s actually not a typo. The list of transforms get replaced with the custom transform, therefore the `D` character does not get converted to its lowercase version and this is why it gets removed.

I should probably make that more clear that in the readme.